website/content/docs/k8s/installation/deployment-configurations/single-dc-multi-k8s.mdx

```diff
@@ -0,0 +1,301 @@
+Set `externalServers.httpsPort` to the `nodePort` of the `cluster1-consul-ui` service.
+In our example, the port is `31557`.
+
+```shell
+$ kubectl get service cluster1-consul-ui --context cluster1
+NAME                 TYPE       CLUSTER-IP    EXTERNAL-IP   PORT(S)         AGE
+cluster1-consul-ui   NodePort   10.0.240.80   <none>        443:31557/TCP   40h
+```
+
+We set the `externalServer.tlsServerName` to `server.dc1.consul`. This the DNS SAN
+(Subject Alternative Name) that is present in the Consul server's certificate.
+We need to set it because we're connecting to the Consul servers over the node IP,
+but that IP isn't present in the server's certificate.
+To make sure that the hostname verification succeeds during the TLS handshake, we need to set the TLS
+server name to a DNS name that *is* present in the certificate.
+
+Next, we need to set `externalServers.k8sAuthMethodHost` to the address of the second Kubernetes API server.
+This should be the address that is reachable from the first cluster, and so it cannot be the internal DNS
+available in each Kubernetes cluster. Consul needs it so that `consul login` with the Kubernetes auth method will work
+from the second cluster.
+More specifically, the Consul server will need to perform the verification of the Kubernetes service account
+whenever `consul login` is called, and to verify service accounts from the second cluster it needs to
+reach the Kubernetes API in that cluster.
+The easiest way to get it is to set it from your `kubeconfig` by running `kubectl config view` and grabbing
+the value of `cluster.server` for the second cluster.
+
+Lastly, we need to set up the clients so that they can discover the servers in the first cluster.
+For this, we will use Consul's cloud auto-join feature
+for the [Kubernetes provider](https://www.consul.io/docs/install/cloud-auto-join#kubernetes-k8s).
+To use it we need to provide a way for the Consul clients to reach the first Kubernetes cluster.
+To do that, we need to save the `kubeconfig` for the first cluster as a Kubernetes secret in the second cluster
+and reference it in the `clients.join` value. Note that we're making that secret available to the client pods
+by setting it in `client.extraVolumes`.
+
+~> **Note:** The kubeconfig you're providing to the client should have minimal permissions.
+The cloud auto-join provider will only need permission to read pods.
+Please see [Kubernetes Cloud auto-join](https://www.consul.io/docs/install/cloud-auto-join#kubernetes-k8s)
+for more details.
+
+Now we're ready to install!
+
+```shell
+helm install cluster2 -f cluster2-config.yaml hashicorp/consul
+```
+
+## Verifying the Consul Service Mesh works
+
+Now that we have our Consul cluster in multiple k8s clusters up and running, we will
+deploy two services and verify that they can connect to each other.
+
+First, we'll deploy `static-server` service in the first cluster:
+
+```yaml
+---
+apiVersion: consul.hashicorp.com/v1alpha1
```

Comment:
@david-yu , @ishustava  Getting 
`no matches for kind "ServiceIntentions" in version "consul.hashicorp.com/v1alpha1"`

Reply:
Hi there. ServiceIntentions are supported when using Consul 1.9 and above, are you perhaps using a Consul 1.9+ image with the Helm values file?

Reply:
Using 1.10 enterprise

Reply:
Hey @arbhatta-blk, just to confirm are you setting `controller.enabled` in your helm configuration?

Reply:
yes. I've set it to true.